Testreglar/1.3.1/Nett/nett-1.3.1b.json

```diff
@@ -0,0 +1,467 @@
+			"kilde": [
+				"H39"
+			],
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.19"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.18"
+				}
+			}
+		},
+		{
+			"stegnr": "3.18",
+			"spm": "Beskriver tabelltittelen innholdet i tabellen?",
+			"ht": "<p>Gjør en skjønnsmessig vurdering av om tabelltittelen eller overskriften beskriver emnet eller formålet med innholdet den hører til.</p>",
+			"kilde": [
+				"H39"
+			],
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"type": "vurderDelutfall",
+							"id": 0,
+							"verdi": "Nei",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Nei",
+								"utfall": "#delutfall(0)"
+							}
+						},
+						"2": {
+							"type": "vurderDelutfall",
+							"id": 0,
+							"verdi": "Ja",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Ja",
+								"utfall": "Tabelltittelen beskriver innholdet i tabellen. #delutfall(0)"
+							}
+						}
+					}
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Tabelltittelen identifiserer ikke innholdet i tabellen. #delutfall(0,Nei)"
+				}
+			}
+		},
+		{
+			"stegnr": "3.19",
+			"spm": "Er tabellen kodet som beskrivende liste?",
+			"ht": "<p>Beskrivende lister (descriptive list) kan ligne på tabell med to koloner, men uten overskriftsceller:</p>\n<ul>\n<li>Sjekk i koden om tabellen du tester er kodet med <code>&lt;dl&gt;</code> og tilhørende <code>&lt;dt&gt;</code>- og <code>&lt;dd&gt;</code>-elementer.</li>\n</ul>\n<p><strong>Merk:</strong> Beskrivende lister skal testes i nett-1.3.1c.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Tabellen er kodet som beskrivende liste."
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Tabellen er ikke korrekt kodet."
+				}
+			}
+		},
+		{
+			"stegnr": "3.20",
+			"spm": "Er tabellen kodet slik at den ignoreres av hjelpemiddelteknologi?",
+			"ht": "<p><strong>Alternativ 1</strong></p>\n<ul>\n<li>Inspiser tabellen i nettleseren</li>\n<li>Bruk Accessibility Checker verktøy i inspiser</li>\n<li>Under Computed Properties i Accessibility Checker\n<ul>\n<li>Sjekk om det står informasjon \"Accessibility node not exposed\"</li>\n</ul>\n</li>\n</ul>\n<p><strong>Alternativ 2</strong></p>\n<p>Sjekk i koden om tabellen er kodet med et av følgende attributter:</p>\n<ul>\n<li><code>role=\"presentation\"</code></li>\n<li><code>role=\"none\"</code></li>\n<li><code>aria-hidden=\"true\"</code></li>\n</ul>\n<p></p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Tabell er kodet slik at den ignoreres av hjelpemiddelteknologi."
```

Comment:
Utfallet "Tabell er kodet slik at den ignoreres av hjelpemiddelteknologi." blir for upresist. Må presisere at dette er en feil for datatabeller / tabeller som inneholder informasjon. (Dette er jo korrekt for layout-tabeller).

Reply:
Tabell til "Datatebell" @Sivzilla82 ser litt på det sammen med siste kommentar.